test/lib/analyzer/image-inspector.spec.ts

```diff
@@ -238,28 +231,17 @@ describe("getImageArchive", () => {
         password,
       );
 
-      expect(dockerPullSpy).toHaveBeenCalled();
-      expect(archiveLocation.path).toEqual(
-        path.join(imageSavePath, "image.tar"),
-      );
-
-      const imageExistsOnDisk: boolean = fs.existsSync(
```

Comment:
But does it mean we no longer create archives on disk? Do we not need to test that? Also do we not need to test that the archive was properly removed after calling `archiveLocation.removeArchive();`?

Reply:
Good shout on removeArchive, I missed that this was set as a callback if the function under test works properly. I think I need to update the test so that the stub implementation creates a dummy file and then add removeArchive() back in, then assert the file gets removed as before.

Reply:
@ivanstanev  updated :)